dev-lang/micropython/micropython-1.23.0.ebuild

```
@@ -0,0 +1,73 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Python implementation for microcontrollers"
+HOMEPAGE="https://github.com/micropython/micropython"
```

_Hello @tfemby,_

Could you please add the official website to the `HOMEPAGE` variable?

```suggestion
HOMEPAGE="https://micropython.org https://github.com/micropython/micropython"
```

_Thanks!_

---

Hey, sorry that I missed that. I added your suggestion :)